qa/tasks/cephfs/test_volume_client.py

```diff
@@ -14,11 +14,12 @@ class TestVolumeClient(CephFSTestCase):
     # One for looking at the global filesystem, one for being
     # the VolumeClient, two for mounting the created shares
     CLIENTS_REQUIRED = 4
-    py_version = 'python'
+    py_version = 'python3'
```

Comment:
Does this change still work on CentOS 7?

Reply:
This one is not unconditional; see - https://github.com/rishabh-d-dave/ceph/blob/make-embdedded-programs-in-test_volume_client-py3-compatible/qa/tasks/cephfs/test_volume_client.py#L21.

Reply:
I don't really see a need to change the default. Instead, use the overrides as we already do.

Reply:
Ok. So is there a way to choose Python version when running vstart_runner.py? Right now I have to do that manually.